tests/test_Summary.cpp

```diff
@@ -182,7 +182,7 @@ struct setup {
         ta( ERT::TestArea("test_summary") ),
         solution( make_solution( es.getInputGrid() ) )
     {
-        solution = make_solution( *es.getInputGrid());
+        solution = make_solution( es.getInputGrid());
```

Comment:
Not really related to this PR, but anyone know why solution isn't just in the init list?